### PR TITLE
update dg 

### PR DESCRIPTION
**Pull request by rachelljt**
- Change the UI model diagram and deleteTask sequence diagram

### CODE REVIEW

File: docs/diagrams/DeleteTaskSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
```
Comment (tashawan23) [inline docs/diagrams/DeleteTaskSequenceDiagram.puml]: AddressBook has been changed to HeyMatez

File: docs/diagrams/DeleteTaskSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("deleteTask 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("deleteTask 1")
```
Comment (tashawan23) [inline docs/diagrams/DeleteTaskSequenceDiagram.puml]: Just need to change these names to HeyMatez

**Review by tashawan23 — CHANGES_REQUESTED** [review]
Looks good! just that AddressBook can be changed to HeyMatez already

File: docs/diagrams/DeleteTaskSequenceDiagram.puml (hunk 1 above)
Comment (rachelljt) [reply docs/diagrams/DeleteTaskSequenceDiagram.puml]: okay!

**Review by tashawan23 — APPROVED** [review]
Looks good!

**Pull request merged**